### ISSUE TITLE
is it possible to do import * as foo from 'blah/*;

### ISSUE THREAD

**Issue by bdwain** [kei-ito/rollup-plugin-glob-import#6]
i noticed that the imports from a glob are not combined into a big object if you use * as. 

```js
import * as foo from 'blah/*.js';
console.log(foo);
```
gets transpiled to
```js
var foo = Object.freeze({

});

console.log(foo);
```

would this be possible? I'm actually trying to say `export * from 'blah/*.js'`, but I think the problem is the same in both cases.

**Comment by kei-ito** [comment]
No. This plugin doesn't support `import X from Y` **now**.
I'll try that this weekend. It may need making rules because there is a number of import syntaxes.

**Comment by bdwain** [comment]
Cool alright thanks!

**Comment by kei-ito** [comment]
I published v0.1.0 and it'll works as you expected.
For example, If you rollup() the [/test/projects/001/src/index.js](https://github.com/kei-ito/rollup-plugin-glob-import/blob/222a05566f565bde8fb93b49f882cf66dc084689/test/projects/001/src/index.js) and run the code, you'll get [the result](https://github.com/kei-ito/rollup-plugin-glob-import/blob/222a05566f565bde8fb93b49f882cf66dc084689/test/projects/001/expected.json).

**Comment by bdwain** [comment]
cool thanks!
